packages/cli/package.json

```diff
@@ -130,7 +130,6 @@
       "@shopify/app",
       "@shopify/theme",
       "@shopify/plugin-did-you-mean",
-      "@shopify/cli-hydrogen",
```

Comment:
why removing hydrogen here?

Reply:
Isn't it not used anymore? I removed it because `oclif readme` was throwing an error saying that it couldn't find the `cli-hydrogen` plugin. It's not part of the CLI anymore right?

Reply:
it's not part of this repo, but it's part of the CLI.
Hydrogen is still a plugin of the CLI, but installed differently, you can see it [here](https://github.com/Shopify/hydrogen/blob/ed0c7e05c4c3c8b49c4829018e584c04217cafec/templates/hello-world/package.json#L16-L17) in their template

So the CLI package needs to know that a hydrogen plugin exists